src/App.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+  const [searchQuery, setSearchQuery] = useState<string>('');
```

Comment:
avoid types for primitives, typescript will do it for you
```suggestion
  const [searchQuery, setSearchQuery] = useState('');
```